### ISSUE TITLE
AAW unit resets constantly on 1.3.0

### ISSUE THREAD

**Issue by bdzirba** [Snirpo/aawireless-issues#62]
The AAW is connecting fine, however after few seconds to few minutes (usualy closer to first option) the unit reboots and connect again. This makes unit unusable. 

Most of the time tried with USB mode: MTP and Direct, however MTP with force/retry as well. Tried passthrough but it seems that it was worse. 
Most of the time "Adjust audio timestamp" as I'm VAG car user, but tried without it as well. 
Wifi timeout was tried with multiple values, starting from 5000, most of the time 12-16000, up to 50k (as one user suggested).

Same phone, same car, same cable - direclty without AAW works fine. 
Did reinstall companion app, no help. 

Send a bug report today (24.05.2021).
Unit name: AAWireless-kw6sIDXK 
Unit version: 1.3.0 (on factory delivered 1.2.3, it was the same).

Skoda Superb III from 2017
Columbus / MIB 2 (with physical buttons)
Samsung Galaxy S21+
Android 11

**Comment by Snirpo** [comment]
If I understand correctly, this same thing happened on 1.2.3?

**Comment by bdzirba** [comment]
Yes, the same issue was on v1.2.3. I received unit from NL with DRAM fix already appliad according to info, and with v1.0.0. The first thing I did was update to 1.2.3. 
Just after release v1.3.0 I followed with update and did some testing again but no change.

**Comment by Snirpo** [comment]
Does AAWireless reboot or the head unit itself?

**Comment by bdzirba** [comment]
Head unit is working fine. 
I see AAW changes from blue solid LED to green solid->blinking... and in the meantime head unit shows that Aandroid Auto device disconnected. Then after ~20s when AAW reboots, AA is working fine again. Few minutes and it reboots again.

**Comment by Snirpo** [comment]
Sorry, I finally had the change to take a look at the logs. The device seems to fully reboot. You have a second batch device, so it could be very well defective. Please create a support ticket at https://cpeb.freshdesk.com/, I'll get you a new device.

**Comment by TadejZupancic** [comment]
> Head unit is working fine. I see AAW changes from blue solid LED to green solid->blinking... and in the meantime head unit shows that Aandroid Auto device disconnected. Then after ~20s when AAW reboots, AA is working fine again. Few minutes and it reboots again.

I have the same issue and also have a second batch device. Did you get a new device? Does it work now?

**Comment by bdzirba** [comment]
I got new device and it solved this issue. Still sometimes there are issues with connectivity but very rare and reboot helps. --Bartosz Dzirba 
-------- Oryginalna wiadomość --------Od: TadejZupancic ***@***.***> Data: 13.01.2022  23:44  (GMT+02:00) Do: cpebit/aawireless-issues ***@***.***> DW: bdzirba ***@***.***>, Author ***@***.***> Temat: Re: [cpebit/aawireless-issues] AAW unit resets constantly (#62) 

Head unit is working fine. I see AAW changes from blue solid LED to green solid->blinking... and in the meantime head unit shows that Aandroid Auto device disconnected. Then after ~20s when AAW reboots, AA is working fine again. Few minutes and it reboots again.

I have the same issue and also have a second batch device. Did you get a new device? Does it work now?

**Comment by TadejZupancic** [comment]
@Snirpo, can you please take a look at my logs from 30.12. if this is the same problem. This has been happening since I've received the device, but I thought it was connected to the VAG problem. But the car display is working fine and the device itself reboots according to the lights. Looks like I also have a defective device from the second batch.

**Comment by block6791** [comment]
Update: CP & EB IT support send me the following steps to resolve the problem:

- Reset the device and unplug it afterwards
- Clear all app data from and uninstall the AAWireless app
- Clear all cache and data from AA and Play services
- Remove the Bluetooth connection from AAWireless
- Remove all phone mirroring/Android/AAWireless history from your car
- Restart your car
- Restart your phone
- Go to the AA settings on your phone and enable developer mode, check add wireless Projection to Settings
- Plug your phone into the car with a cable and allow it to connect and run through the initial settings
- Unplug your phone
- Plug in the AAWireless unit
- Download and install the AAWireless app again and go through its initial connection again

I followed these instructions in this order and the results are positive. The unit now connects every time and the connections stays on during my commutes. 

Another thing I did is replace the standard AAwireless USB cable. I tried the supplied cable on different devices and discovered it does not connect. For others who have this problem or similar ones, I recommend to first try another known good USB cable before you try anything else. For me it was the combination of the steps above and the cable replacement that solved it.
